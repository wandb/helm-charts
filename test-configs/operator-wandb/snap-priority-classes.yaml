global:
  bucket:
    provider: "s3"
    name: "minio.minio.svc.cluster.local:9000/bucket"
    region: "us-east-1"
    accessKey: "minio"
    secretKey: "testpassword"
  env:
    GLOBAL_ADMIN_API_KEY: "local-123456789-123456789-123456789-1234"
    GORILLA_INSECURE_ALLOW_API_KEY_ADMIN_ACCESS: "true"
  size: "testing"
  # Global priority class - should apply to all components that don't have specific config
  priorityClassName: "wandb-global-priority"
  nodeSelector:
    environment: "testing"
  tolerations:
    - effect: NoSchedule
      key: "testing"
      operator: Equal
      value: "true"

app:
  env:
    GORILLA_STORAGE_BUCKET: "s3://local-files"

# Console with component-specific priority class override
console:
  priorityClassName: "wandb-console-priority"
  containers:
    console:
      resources:
        requests:
          cpu: 50m
          memory: 64Mi

# API without specific priority class - should inherit global
api:
  containers:
    api:
      resources:
        requests:
          cpu: 50m
          memory: 64Mi

# Parquet with specific priority class
parquet:
  priorityClassName: "wandb-data-priority"
  containers:
    parquet:
      resources:
        requests:
          cpu: 50m
          memory: 64Mi

ingress:
  install: false
  create: false

mysql:
  install: true
  # MySQL should use its own priorityClassName override from the global wandb.priorityClassName helper
  priorityClassName: "wandb-database-priority"
  resources:
    requests:
      cpu: "50m"
      memory: "64Mi"

redis:
  install: true
  resources:
    requests:
      cpu: "50m"
      memory: "64Mi"

reloader:
  install: true
  resources:
    requests:
      cpu: "50m"
      memory: "64Mi"

anaconda2:
  install: true
