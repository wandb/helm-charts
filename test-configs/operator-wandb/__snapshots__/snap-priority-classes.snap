apiVersion: helm-chartsnap.jlandowner.dev/v1alpha1
kind: Unknown
metadata:
  name: helm-output
raw: |
  Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, wandb-base, reloader
